frontend/src/features/projects/common/projectConverter.ts

```diff
@@ -208,15 +211,19 @@ export const toApiProperty = (property: IProperty): IApiProperty => {
         parcelId: property.propertyTypeId === 0 ? property.id : undefined,
         buildingId: property.propertyTypeId === 1 ? property.id : undefined,
         value: property.netBook,
-        fiscalYear: property.netBookFiscalYear ?? getCurrentFiscalYear(),
+        fiscalYear: !useCurrentFiscal
+          ? property.netBookFiscalYear ?? getCurrentFiscalYear()
```

Comment:
What does `getCurrentFiscalYear()` return?

Reply:
Do you need something similar for `Evaluations`?

Reply:
The evaluation are working just fine

Reply:
> What does `getCurrentFiscalYear()` return?

@Fosol returns the current fiscal year